docs/application/native/api/iot-headed/5.0/group__Evas__Main__Group.html

```diff
@@ -98,13 +98,17 @@
 <tr><td colspan="2"><h2><a name="func-members"></a>
 Functions</h2></td></tr>
 <tr><td class="memItemLeft" align="right" valign="top">int&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="group__Evas__Main__Group.html#ga65b3f060b6dd9e488afd0b8cbe101869">evas_init</a> (void)</td></tr>
```

Comment:
Need to check the function 'evas_init (void)' compared to mobile API Reference.
(ref. https://docs.tizen.org/application/native/api/mobile/6.0/group__Evas__Main__Group.html : EINA_DEPRECATED int)

Reply:
It is wrong deprecated TAG. I've added an issue to https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=194846376
Issue No. 60

EFL part need to update header file.... Can't fix it right now.

Reply:
Okay, I got it.